solr/core/src/test/org/apache/solr/handler/TestReplicationHandlerDiskOverFlow.java

```diff
@@ -67,6 +67,7 @@
 
   @Before
   public void setUp() throws Exception {
+    systemSetPropertySolrDisableShardsWhitelist("true");
```

Comment:
same question here, re: deprecation

Reply:
This is also for backward compatibility for an existing test. The same pattern is used for 10 other existing tests.
Maybe I can try to replace that by a dynamic solr config in the 9x patch, but should we do that here?

Reply:
Finally I could remove this particular use of the deprecated method in favour of settings the whitelist (which is not possible in TestReplicationHandler because it creates JettySolrRunner later so we don't know the ports ahead of time).